west.yml

```diff
@@ -148,6 +148,9 @@ manifest:
     - name: nanopb
       path: modules/lib/nanopb
       revision: d148bd26718e4c10414f07a7eb1bd24c62e56c5d
+    - name: lz4
+      path: modules/lib/lz4
+      revision: 07b1b43b7877cfbc57ed667652ad18fff83cc637
```

Comment:
https://github.com/zephyrproject-rtos/lz4/commit/8e303c264fc21c2116dc612658003a22e933124d that's the current HEAD? So the revision is 8e303c264fc21c2116dc612658003a22e933124d?